src/ACESVGjsPanel.tsx

```diff
@@ -224,17 +236,16 @@ export class ACESVGPanel extends PureComponent<Props, PanelState> {
       return null;
     }
   }
+
   render() {
+    const { options, width, height } = this.props;
```

Comment:
this is a bit more idomatic -- means you don' tneed `this.props.` as the prefix below